l10n_br_fiscal_certificate/models/res_company.py

```diff
@@ -44,3 +46,10 @@ def _compute_certificate(self):
                 )
 
             record.certificate = certificate
+
+    def _get_br_ecertificate(self):
```

Comment:
como esse método não depende dos registros, ele deve ser @api.model

Reply:
valeu @rvalyi , obrigado pela revisão, alteração feita

Reply:
Será que não seria melhor deixar mais explícito no nome do método que ele serve para instanciar o certificado na biblioteca `erpbrasil`? Talvez usar o nome `_get_erpbrasil_certificate_instance`? É que, caso contrário, acho que fica confuso distinguir do `self.certificate`.

Reply:
Valeu @antoniospneto , obrigado pela revisão, acredito que isso tem relação com o que escrevi sobre avaliar a necessidade de manter o campo compute ou integrar tudo nesse método, já que com a alteração do PR apenas o teste do modulo ainda vai estar usando o self.certificate, a questão é:

Sempre que for usado o Certificado isso já deve ser devolvido instanciado? Ou existiria alguma situação/processo em que vai ser necessário ter apenas o Certificado, o valor que é devolvido pelo campo compute self.certificate?

Porque se sempre vai ser devolvido dentro de uma instancia então só vai existir um método aí sem margem para dúvidas, com isso o nome pode ser tanto um como outro